importers/tuition-fees-importer.php

```diff
@@ -262,40 +271,48 @@ private function update_degrees() {
 		}
 	}
 
-	private function get_schedule_code( $program_type, $plan_code, $subplan_code, $is_online ) {
+	private function get_schedule_code( $degree, $program_type, $plan_code, $subplan_code, $is_online ) {
```

Comment:
It looks like most of the logic in this script is dependent on `$program_type` having a value. We should do a null check early on in the function and short circuit the function if it is.

Reply:
re. both comments:  We do a short circuit on line 251 and prevent any null `$program_type`'s from getting passed into `get_schedule_code()` to begin with.  We could add an extra `$program_type` check in `get_schedule_code()` for extra hardening if we wanted to, but in its current state, we shouldn't ever see any warnings thrown due to `$program_type` being null.

Reply:
Ok. As long as `get_schedule_code` stays private, and isn't used anywhere else in the class, it should be fine. If we ever think we might want to make it public, adding a null check will make it a little safer for use.